src/actions.ts

```diff
@@ -158,3 +158,39 @@ When('I hover over {string}', async function (alias: string) {
     await conditionWait(element, conditionValidations.VISIBLE, config.browser.timeout.visible);
     await element.moveTo();
 });
+
+/**
+ * Select option with certain text from select element
+ * @param {string} option - option to select
+ * @param {string} alias - alias of select
+ * @example I select '1900' option from 'Registration Form > Date Of Birth'
+ * @example I select '$dateOfBirth' option from 'Registration Form > Date Of Birth'
+ */
+When('I select {string} option from {string} select', async function (option: string, alias: string) {
```

Comment:
maybe it is better to rename select to dropdown

Reply:
make sense

Reply:
updated